sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/_azureappconfigurationprovider.py

```diff
@@ -573,7 +598,7 @@ def refresh(self, **kwargs) -> None:
                 except AzureError as e:
                     exception = e
```

Comment:
We need to log something when failover occurs. Currently, how does a user know when we failover automatically?

Reply:
In .NET we log warnings. Something like - "Failed to refresh from endpoint A. Failing over to endpoint B" 
This helps users track their failures through logs.

Reply:
I can do the first part. I abstracted out the failover process, so we don't know in this part of the code if they have other endpoints. Does just saying `logger.warn("failed to refresh from endpoint " + client.endpoint) work`?

Reply:
I see that we don't have any log statements in the current code. Is that something we are tracking as a separate work item?

Reply:
Ok, sounds good.

Btw, we initialize a logger here but never use it: https://github.com/Azure/azure-sdk-for-python/blob/947ad7cdf83f90fa82db28ed7e2726d5d12cbcbe/sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/_azureappconfigurationprovider.py#L61